.vale/fixtures/Localization/WordswithMultipleMeanings/testinvalid.adoc

```diff
@@ -2,4 +2,7 @@ as
 at once
 once
 while
-since
\ No newline at end of file
+since
+as soon as
```

Comment:
The as soon as onwards should actually be in the Valid file. We want Vale to catch "as" on its own, but not expressions like "as much as" or "as soon as"